packages/l/libsdl_image/xmake.lua

```diff
@@ -38,7 +52,11 @@ package("libsdl_image")
     on_install(function (package)
         local configs = {"-DSDL2IMAGE_SAMPLES=OFF", "-DSDL2IMAGE_TESTS=OFF"}
         table.insert(configs, "-DCMAKE_BUILD_TYPE=" .. (package:debug() and "Debug" or "Release"))
-        table.insert(configs, "-DBUILD_SHARED_LIBS=" .. (package:config("shared") and "ON" or "OFF"))
+        if package:version():le("2.6.2") then
+            table.insert(configs, "-DBUILD_SHARED_LIBS=" .. (package:config("shared") and "ON" or "OFF"))
+        else
+            table.insert(configs, "-DPNG_SHARED=" .. (package:config("shared") and "ON" or "OFF"))
```

Comment:
why? png? no this option, only `SDL2IMAGE_PNG_SHARED`

we should use BUILD_SHARED_LIBS

https://github.com/libsdl-org/SDL_image/blob/36cbe2624d233e5cbc5d88427e0f6b0d4219caa2/CMakeLists.txt#L68

Reply:
oh, then i just got confused by https://github.com/libsdl-org/SDL_image/compare/release-2.6.2...release-2.6.3#diff-1e7de1ae2d059d21e1dd75d5812d5a34b0222cef273b7c3a2af62eb747f9d20aL488-R488